gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/FsJobStatusRetriever.java

```diff
@@ -135,32 +135,6 @@ public FsJobStatusRetriever(Config config) {
     }
   }
 
-  /**
-   *
-   * @param jobState instance of {@link State}
-   * @return deserialize {@link State} into a {@link JobStatus}.
-   */
-  private JobStatus getJobStatus(State jobState) {
```

Comment:
did u mean to remove this method? its still being used in same class.

Reply:
I moved it back to the parent class, because I need it in other child class of its parent class.